src/main/python/janusgraph_python/core/datatypes/Circle.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Circle(object):
+    def __init__(self, longitude, latitude, radiusInKM):
+        """
+            The actual class representing Geographical Circle.
+
+        Args:
+            latitude (float): The latitude of circle's center.
+            longitude (float): The longitude of circle's center
+            radiusInKM (int): The radius of circle in KM.
+        """
+
+        self.latitude = float(latitude) * 1.0
+        self.longitude = float(longitude) * 1.0
+        self.radius = radiusInKM
+
+        status = self.__are_valid_coordinates()
+
+        if not status:
```

Comment:
(nitpick) Here, I would again remove the temp variable `status`. Reading `if not status` doesn't tell me anything. I still have to read the line above to understand this. If it would be simply `if not self.__are_valid_coordinates()` (or even better `if self._are_coordinates_invalid` to avoid the negation), then I would directly understand the `if` clause.

Reply:
@FlorianHockmann : Haha. Nice catch. Really makes it more readable now :-)